deploy/clowdapp.yaml

```
@@ -104,7 +106,7 @@ objects:
                 cpu: ${CPU_REQUESTS}
                 memory: ${MEMORY_REQUESTS}
         - name: statuser
-          # statuser pod must have exactly one replicas
+          # statuser pod is just one replica for easier throttling
           replicas: 1
```

I'd love to try the annotations here, but up to you :)
```suggestion
          replicas: 1
          metadata:
            annotations:
              ignore-check.kube-linter.io/minimum-three-replicas: "statuser pod runs in a single instance"
```

---

Let’s try: https://github.com/RHEnVision/provisioning-backend/pull/467

Let’s merge whatever works.